client_admin/pulp/client/admin/tasks.py

```diff
@@ -231,12 +247,27 @@ def retrieve_tasks(self):
 class AllTasksSection(BaseTasksSection):
     FIELDS = ('tags', 'task_id', 'state', 'start_time', 'finish_time')
 
+    def __init__(self, context, name, description):
+        BaseTasksSection.__init__(self, context, name, description)
+
+        self.list_command.add_option(self.all_flag)
```

Comment:
if i will add them right after the the list command is created(in BaseTasksSection) it will end up that RepoTasksSection will also have those options because it inherits the BaseTasksSection